translations/pl.json

```diff
@@ -23,9 +23,9 @@
 	"--kvm-numa-count range is 1-8": "",
 	"--network flag is only valid with the docker/podman and KVM drivers, it will be ignored": "",
 	"\u003ctarget file absolute path\u003e must be an absolute Path. Relative Path is not allowed (example: \"/home/docker/copied.txt\")": "",
-	"==\u003e Audit \u003c==": "",
-	"==\u003e Last Start \u003c==": "",
-	"A VPN or firewall is interfering with HTTP access to the minikube VM. Alternatively, try a different VM driver: https://minikube.sigs.k8s.io/docs/start/": "",
+	"==\u003e Audit \u003c==": "==> Audyt <==",
```

Comment:
Lets leave the symbols as escaped unicode to ensure no inconsistencies
`==\u003e Audyt \u003c==`